code/game/objects/items/dehy_carp.dm

```diff
@@ -6,22 +6,21 @@
 // Child of carpplushie because this should do everything the toy does and more
 /obj/item/toy/carpplushie/dehy_carp
 	var/mob/owner = null	// Carp doesn't attack owner, set when using in hand
-	var/owned = 1	// Boolean, no owner to begin with
+	var/owned = 0	// Boolean, no owner to begin with
 
 // Attack self
 /obj/item/toy/carpplushie/dehy_carp/attack_self(mob/user)
 	src.add_fingerprint(user)	// Anyone can add their fingerprints to it with this
-	if(owned)
-		user << "<span class='notice'>[src] stares up at you with friendly eyes.</span>"
+	if(!owned)
+		user << "<span class='notice'>You pet [src]. You swear it looks up at you.</span>"
 		owner = user
-		owned = 0
+		owned = 1
 	return ..()
 
 
 /obj/item/toy/carpplushie/dehy_carp/afterattack(obj/O, mob/user,proximity)
 	if(!proximity) return
 	if(istype(O,/obj/structure/sink))
```

Comment:
nope https://github.com/tgstation/-tg-station/blob/b2948fcab7d2be5ce99210fcf5679cc9a886f48f/code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

really what I should do is kill the sink message and put it in the swell proc but it's late